src/overlays/actors/ovl_Bg_Spot17_Bakudankabe/z_bg_spot17_bakudankabe.c

```diff
@@ -6,6 +6,17 @@
 
 #include "z_bg_spot17_bakudankabe.h"
 #include "overlays/effects/ovl_Effect_Ss_Kakera/z_eff_ss_kakera.h"
+
+#include "libc64/qrand.h"
+#include "gfx.h"
+#include "gfx_setupdl.h"
+#include "ichain.h"
+#include "sfx.h"
+#include "sys_matrix.h"
```

Comment:
```suggestion
```
(what's your workflow for determining which headers to include?)
(I'm not gonna check everything there's too much, I'll find a way to clean it up in a subsequent pr)